### PR TITLE
Fix designs

### PR DESCRIPTION
**Pull request by aqla114**
画面のデザインを完成させていく。

### CODE REVIEW

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by aqla114** [comment]
TopScene
image

**Comment by aqla114** [comment]
#99 
StageSelectingScene
image

**Comment by aqla114** [comment]
PlayingScene
image

**Comment by aqla114** [comment]
GameOverScene
image

**Comment by aqla114** [comment]
ResultScene
image

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by aqla114** [comment]
#122 エクセレントクリア実装した。

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

File: src/stages.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export type Stage = {
 	description: string;
 	characterInitialPosition: CharacterPosition;
 	clearPoint: number;
+	excellentClearNroma: number;
```
Comment (turenar) [inline src/stages.ts]: norma?

**Review by turenar — APPROVED** [review]

File: src/stages.ts (hunk 1 above)
Comment (aqla114) [reply src/stages.ts]: yes norma

**Comment by ghost** [comment]
deployed! https://turenar.xyz/mayfes-pikyou/fix-designs

**Pull request merged**